### ISSUE TITLE
Integration with NetBeans formatting options

### ISSUE THREAD

**Issue by cdalexndr** [welovecoding/editorconfig-netbeans#124]
Instead of some _magic_ that happens when you save your file, it would be better to allow to load the formatting options from an `.editorconfig` file directly into NetBeans formatting options (overwriting them). This way formatting will also work with manual format (Alt+Shit+F).

**Comment by Chris2011** [comment]
I don't think that each config property inside editorconfig, can map to the options, due to limitations for other languages than Java. So it would be better to fix the plugin IMHO.

**Comment by bennycode** [comment]
@Chris2011 thanks for keeping the topic alive. 

@Yserz and I haven't been working on the NetBeans EditorConfig plugin for years. We both are fulltime TypeScript developers now and work most of the times with WebStorm and Visual Studio Code. That's why we deprecated the NetBeans EditorConfig plugin.

I just added a [deprecation notice](https://github.com/welovecoding/editorconfig-netbeans/commit/2796df7e1756b7212c89b20f39b3b0f489d6c38f) and linked to the [ticket on the Apache's JIRA board](https://issues.apache.org/jira/browse/NETBEANS-1287).

**Comment by Chris2011** [comment]
I think the biggest problems are those bugs: #109, #110 and #111. if those could be fixed, we can think of transfering the code to the Apache NetBeans core :). But before I don't think so.
